packages/cicero-cli/index.js

```diff
@@ -24,480 +24,596 @@ require('yargs')
     .demandCommand(1, '# Please specify a command')
     .recommendCommands()
     .strict()
-    .command('parse', 'parse a contract text', (yargs) => {
-        yargs.option('template', {
-            describe: 'path to the template',
-            type: 'string'
-        });
-        yargs.option('sample', {
-            describe: 'path to the contract text',
-            type: 'string'
-        });
-        yargs.option('output', {
-            describe: 'path to the output file',
-            type: 'string'
-        });
-        yargs.option('currentTime', {
-            describe: 'set current time',
-            type: 'string',
-            default: null
-        });
-        yargs.option('utcOffset', {
-            describe: 'set UTC offset',
-            type: 'number',
-            default: null
-        });
-        yargs.option('offline', {
-            describe: 'do not resolve external models',
-            type: 'boolean',
-            default: false
-        });
-        yargs.option('warnings', {
-            describe: 'print warnings',
-            type: 'boolean',
-            default: false
-        });
-    }, (argv) => {
-        if (argv.verbose) {
-            Logger.info(`parse sample ${argv.sample} for template ${argv.template}`);
-        }
+    .command(
```

Comment:
Changes in the formatting make it really hard to understand the change... I would generally try and avoid them (unless the PR is purely formatting change with no new feature).